runtime/j9vm/java11vmi.c

```diff
@@ -550,10 +560,17 @@ getJ9Module(J9VMThread * currentThread, jobject module)
 }
 
 static BOOLEAN
-isModuleJavaBase(j9object_t moduleName)
+isModuleJavaBase(J9VMThread * currentThread, j9object_t moduleName)
 {
-	/** @todo compare against string 'java.base' */
-	return FALSE;
+	PORT_ACCESS_FROM_VMC(currentThread);
+	char moduleNameBuffer[J9VM_PACKAGE_NAME_BUFFER_LENGTH];
+	char *moduleNameBufferUTF = currentThread->javaVM->internalVMFunctions->copyStringToUTF8WithMemAlloc(
+		currentThread, moduleName, J9_STR_NULL_TERMINATE_RESULT, "", 0, moduleNameBuffer, J9VM_PACKAGE_NAME_BUFFER_LENGTH, NULL);
+	BOOLEAN result = !strcmp("java.base", moduleNameBufferUTF);
```

Comment:
Can you make this an explicit boolean condition?  ie: `== 0 or != 0`

Reply:
See Tobi's suggestion about `compareStringToUTF8`.